src/encoder/index.js

```diff
@@ -36,7 +36,7 @@ export const encodeNumber = (num) => UVarInt.encode(num)
  * encode bool
  * @param b
  */
-export const encodeBool = (b) => b ? VarInt.encode(1): VarInt.encode(0)
+export const encodeBool = (b) => b ? UVarInt.encode(1): UVarInt.encode(0)
```

Comment:
This was wrong because we do not have a `bool` in any of the existing msg types supported by sdk?

Reply:
right.